babel.config.js

```diff
@@ -0,0 +1,4 @@
+// babel.config.js
+module.exports = {
```

Comment:
Is there a reason we need this, considering it is in our webpack config?  Just curious, no change requested.

Reply:
Good question. I will remove this in the next pull request if JEST does not create any issues.